metamorph/store/postgresql/postgres.go

```diff
@@ -464,6 +478,10 @@ func (p *PostgreSQL) GetUnmined(ctx context.Context, since time.Time, limit int6
 			data.LockedBy = lockedBy.String
 		}
 
+		if retries.Valid {
+			data.Retries = int(retries.Int32)
+		}
+
 		err = p.setLockedBy(ctx, data.Hash, p.hostname)
```

Comment:
We should set the locked by field only to the hostname if it is not set yet. That may save us some unnecessary updates on the DB
```
		if data.LockedBy == "NONE" {
			err = p.setLockedBy(ctx, data.Hash, p.hostname)
			if err != nil {
				return nil, err
			}
		}
```